docs/documentation/functions/aria-suite/aria-operations-networks/Update-AriaNetworksvCenterDataSourceCredentials.md

```diff
@@ -0,0 +1,128 @@
+# Update-AriaNetworksvCenterDataSourceCredentials
+
+## Synopsis
+
+Update a configured vCenter Server data source's credentials in VMware Aria Operations for Networks.
+
+## Syntax
+
+```powershell
+Update-AriaNetworksvCenterDataSourceCredentials [-id] <String> [-collectorId] <String> [[-fqdn] <String>]
+ [-nickname] <String> [-username] <String> [-password] <String> [<CommonParameters>]

```

Comment:
```suggestion
Update-AriaNetworksvCenterDataSourceCredentials [-id] <String> [-collectorId] <String> [[-fqdn] <String>] [-nickname] <String> [-username] <String> [-password] <String> [<CommonParameters>]
```

Reply:
Fixed in https://github.com/vmware/power-validated-solutions-for-cloud-foundation/commit/7344af3d25523db031599dbdd47c90a7d78ba0d9